### PR TITLE
Update contribution guidelines

### PR DESCRIPTION
**Pull request by alina-beck**
## 🍰 Pullrequest
I actually just wanted to add the communication guidelines we came up with a few weeks ago but then felt the whole `contribution readme` was rather outdated... So here's an update!

### Issues
- None (and here I am violating one of the guidelines, yay! 😅)

### Todo
- [ ] please, please, please check if I got all the times right!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,79 +1,99 @@
...
 
 ## Getting Set Up
 
-Instructions for how to install all the necessary software can be found in our [documentation](https://docs.human-connection.org/human-connection/).
+Instructions for how to install all the necessary software and some code guidelines can be found in our [documentation](https://docs.human-connection.org/human-connection/).
 
-We recommend that new folks should ideally work together with an existing developer. Please join our [discord](https://discord.gg/6ub73U3) instance to chat with developers or just ask them in tickets in [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f/boards?repos=152252353):
+To get you started we recommend that you join forces with a regular contributor. Please join [our discord instance](https://discord.gg/6ub73U3) to chat with developers or just get in touch directly on an issue on either [Github](https://github.com/Human-Connection/Human-Connection/issues) or [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f/boards?repos=152252353):
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
To get you started we recommend that you join forces with a regular contributor. Please join [our discord instance](https://human-connection.org/discord) to chat with developers or just get in touch directly on an issue on either [Github](https://github.com/Human-Connection/Human-Connection/issues) or [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f/boards?repos=152252353):
```

Comment (roschaefer) [reply CONTRIBUTING.md]: For reasons, the discord invite link got lost sometimes in the past. This redirect should always lead to a valid invite link.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,79 +1,99 @@
...
+3. We make sure we understand the issue in detail – what problem is it solving and how should it be implemented?
+4. We assign ourselves to the issue and move it to `In Progress` on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f).
+5. We start working on it in a `new branch` and open a `pull request` prefixed with `[WIP]` (work in progress) to which we regularly push our changes.
+6. When questions come up we clarify them with the team (directly in the issue on Github).
+7. When we are happy with our work and our PR is passing all tests we remove the `[WIP]` from the PR description and ask for reviews (if you're not sure who to ask there is @Human-Connection/hc-dev-team which pings all core developers).
+8. We then incorporate the suggestions from the reviews into our work and once it has been approved it can be merged into master!
+
+Every pull request needs to:
+* fix an issue (if there is something you want to work on but there is no issue for it, create on first and discuss it with the team)
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
* fix an issue (if there is something you want to work on but there is no issue for it, create one first and discuss it with the team)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,79 +1,99 @@
...
+* Wolle (@Tirokk)
+* Alex (@ogerly)
+* Alina (@alina-beck)
+* Martin (@datenbrei), our head of IT
+* and sometimes Dennis (@DennisHack), the founder of Human Connection
+
+## Meetings and Pair Programming Sessions
+
+Daily standup
+* every Monday–Friday 10:45 UTC
```
Comment (roschaefer) [inline CONTRIBUTING.md]: Publishing those events on UTC means it's a reference. So, if Germany has its clocks shifted from CET to CEST, our meeting in local time will change from 11:45 CET to 10:45 CEST. Are we OK with that? @mattwr18 @datenbrei @Tirokk @ogerly @alina-beck ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,79 +1,99 @@
...
+* everybody shares what they are working on and asks for help if they are blocked
+
+Regular pair programming sessions
+* every Monday, Wednesday and Thursday 14:00 UTC
+* the link will be posted in the [discord chat](https://discord.gg/6ub73U3) and on the [Agile Ventures website](https://www.agileventures.org/events?utf8=%E2%9C%93&project_id=220&commit=Filter+by+Project)
+* all contributors welcome!
+* we team up and work on an issue together (often using Visual Studio live sharing sessions)
+
+Developer's Community Meeting
+* every Thursday 12:00 UTC
```
Comment (roschaefer) [inline CONTRIBUTING.md]: When I see English text and a 12:00 meeting I always wonder: 12:00am, 12:00pm? What's actually 12:00pm? Noon, midnight? :thinking: 

From the context I can tell that you are using 24 hours clock here (which I prefer for the reasons above :point_up: ) but it always gives me headache.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,79 +1,99 @@
...
+* the link will be posted in the [discord chat](https://discord.gg/6ub73U3) and on the [Agile Ventures website](https://www.agileventures.org/events?utf8=%E2%9C%93&project_id=220&commit=Filter+by+Project)
+* all contributors welcome!
+* ???
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,79 +1,99 @@
...
+* all contributors welcome!
+* everybody shares what they are working on and asks for help if they are blocked
+
+Regular pair programming sessions
+* every Monday, Wednesday and Thursday 14:00 UTC
+* the link will be posted in the [discord chat](https://discord.gg/6ub73U3) and on the [Agile Ventures website](https://www.agileventures.org/events?utf8=%E2%9C%93&project_id=220&commit=Filter+by+Project)
+* all contributors welcome!
+* we team up and work on an issue together (often using Visual Studio live sharing sessions)
+
+Developer's Community Meeting
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
Open-Source Community Meeting
```
It's a little more inclusive. Sometimes people want to join who are no developers.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,79 +1,99 @@
...
+* every Thursday 12:00 UTC
+* the link will be posted in the [discord chat](https://discord.gg/6ub73U3) and on the [Agile Ventures website](https://www.agileventures.org/events?utf8=%E2%9C%93&project_id=220&commit=Filter+by+Project)
+* all contributors welcome!
+* ???
+
+Meet the team
+* every Monday 20:00 UTC (at the moment only in German)
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
* every Monday 20:00 UTC (at the moment only in German)
* details here https://human-connection.org/veranstaltungen/
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,79 +1,99 @@
...
-  - but often it will be the core developers (Robert, Wolfgang, Matt, Alina, Alex)
-  - once there is a review, and presuming no requested changes, PR opener can merge
-
-* CI/tests
-  - the CI needs to pass
-    - linting (yarn lint --fix)
-    - tests (unit, feature) (backend, frontend)
-    - codecoverage
-
-## Notes
```
Comment (roschaefer) [inline CONTRIBUTING.md]: Stange that these notes found their way into `CONTRIBUTING.md`

File: CONTRIBUTING.md (hunk 3 above)
Comment (datenbrei) [reply CONTRIBUTING.md]: Yes. Also I'd add the place to the standup information (Discord Voicechat -> Coneference Room)

Comment (alina-beck) [reply CONTRIBUTING.md]: Times are now indicated in "german time" – it's not perfect but clear enough, I hope?

File: CONTRIBUTING.md (hunk 4 above)
Comment (alina-beck) [reply CONTRIBUTING.md]: It's sooo confusing... luckily we don't have any meetings at 12:00 German time! 😝

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,79 +1,101 @@
...
+* Alina (@alina-beck)
+* Martin (@datenbrei), our head of IT
+* and sometimes Dennis (@DennisHack), the founder of Human Connection
+
+## Meetings and Pair Programming Sessions
+
+Times below refer to **German Time** – that's CET (GMT+1) in winter and CEST (GMT+2) in summer – because most Human Connection core team members are living in Germany.
+
+Daily standup
+* every Monday–Friday 11:45
```
Comment (roschaefer) [inline CONTRIBUTING.md]:
```suggestion
* every Monday–Friday 11:30
```

Comment (alina-beck) [reply CONTRIBUTING.md]: nooo, daily is at 11:45 – or am I just always 15 minutes late? 😜

Comment (mattwr18) [reply CONTRIBUTING.md]: we agreed to change it since we are always rushing to get to the 12pm meeting @alina-beck

File: CONTRIBUTING.md (hunk 4 above)
Comment (mattwr18) [reply CONTRIBUTING.md]: 12pm is midday... 12am is midnight :stuck_out_tongue:

File: CONTRIBUTING.md (hunk 9 above)
Comment (alina-beck) [reply CONTRIBUTING.md]: oh, I had no idea! I'll merge this now, thanks for updating :)

**Pull request merged**